deploy/replication_controller.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: gitservice-devopsconsole-apiserver
+  namespace: openshift-devops-console
+  labels:
+    apiserver: "true"
+spec:
+  replicas: 1
+  selector:
+    apiserver: "true"
+  template:
+    metadata:
+      labels:
+        apiserver: "true"
+    spec:
+      serviceAccountName: apiserver
+      containers:
+      - name: gitservice-devopsconsole-apiserver
+        # Replace this with the built image name
+        image: REPLACE_IMAGE
+        imagePullPolicy: Always
+        command: [ "/git-service", "--etcd-servers=http://localhost:2379" ]
```

Comment:
just curious, did we verify that etcd servers have valid certificates to talk using localhost on openshift 4. Also we need to figure out how can we get HA node etcd clusters endpoints?

Reply:
I'm not sure we really care about that right now. We do not manipulate any resources (at least not yet). Will need to look at this later if needed.